lib/cstring.h

```diff
@@ -165,6 +166,12 @@ class cstring {
     const char *c_str() const { return str; }
     operator const char *() const { return str; }
 
+    std::string string() const { return std::string(str); }
```

Comment:
`str` is the name of the (`private`) member that holds the actual `const char *` data. So that would require renaming it. Also I don't like `.sv()` as a `.string_view()` and if we have `string_view()` it is more consistent to have `.string()`. Naming is hard...